EventVisualisation/DataConverter/include/EventVisualisationDataConverter/VisualisationTrack.h

```diff
@@ -53,21 +55,22 @@ class VisualisationTrack
   /// Simplifies passing parameters to constructor of VisualisationTrack
   /// by providing their names
   struct VisualisationTrackVO {
+    float time;
     int charge;
-    double energy;
-    int ID;
+    //  double energy;
+    //  int ID;
     int PID;
-    double mass;
-    double signedPT;
+    //  double mass;
+    //  double signedPT;
     double startXYZ[3];
-    double endXYZ[3];
-    double pxpypz[3];
-    int parentID;
+    //  double endXYZ[3];
+    //  double pxpypz[3];
+    //  int parentID;
     double phi;
     double theta;
-    double helixCurvature;
-    int type;
-    ETrackSource source;
+    //  double helixCurvature;
+    //  int type;
+    o2::dataformats::GlobalTrackID::Source source;
   };
   // Constructor with properties initialisation
   VisualisationTrack(const VisualisationTrackVO vo);
```

Comment:
pass by reference

Reply:
it is uint8 - why reference? It is value object used to construct VisualisationTrack - so I guess should contains values